core/dict.class.inc.php

```diff
@@ -205,19 +223,36 @@ public static function S($sStringCode, $sDefault = null, $bUserLanguageOnly = fa
 	 */
 	public static function Format($sFormatCode /*, ... arguments ....*/)
 	{
-		$sLocalizedFormat = self::S($sFormatCode);
+		$aInfo = self::GetLabelAndLangCode($sFormatCode);
+		if (array_key_exists('label', $aInfo)){
+			$sLocalizedFormat = $aInfo['label'];
+		} else {
+			$sLocalizedFormat = $sFormatCode;
+		}
+
+		if (array_key_exists('lang', $aInfo)){
+			$sLangCode = $aInfo['lang'];
+		} else {
+			$sLangCode = null;
+		}
```

Comment:
The keys are always existing..
```suggestion
		['label' => $sLocalizedFormat, 'lang' => $sLangCode] = self::GetLabelAndLangCode($sFormatCode);
```

Reply:
+1